interfaces/builtin/content.go

```diff
@@ -165,16 +167,18 @@ func mountEntry(plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot,
 	options := make([]string, 0, len(extraOptions)+1)
 	options = append(options, "bind")
 	options = append(options, extraOptions...)
+	var target string
+	_ = plug.Attr("target", &target)
```

Comment:
maybe worth a comment that this was checked in Sanitize, the old code would have paniced instead